frontend/src/pages/Home.tsx

```diff
@@ -12,45 +12,99 @@ import FAQ from "../components/FAQ";
 import Showcase from "../components/Showcase";
 import Features from "../components/Features";
 import Category from "./Category";
+import { useEffect, useState } from "react";
+import { motion, AnimatePresence } from "framer-motion";
 
 function Home() {
   console.log("Home page rerendered");
   const { t } = useTranslation();
 
-  document.title='Style Share | Welcome 🙏'
-  
+  document.title = 'Style Share | Welcome 🙏';
+
+  // Define the type for visibleItems as an array of strings

```

Comment:
Could we remove this comment ? Code is readable so this is unnecessary here.

Reply:
Sure sir ,Done!